docs/QUICKSTART.md

```diff
@@ -0,0 +1,72 @@
+## Introduction
+
+This guide explains on how to setup the infrastructure to use and test the KUKSA Companion App.
+
+## Requirements
+
+- Android Device with API Level >= 27 (Android 8.1)
+- a running DataBroker
+- a running MockService (Optional)
+
+## Setup Databroker
+
+To use the Companion App a connection to a Databroker is required. This Databroker might run inside
+a car or on your local system for testing purposes. The steps to set up a local Databroker can be
+found in
+the [KUKSA.val Quickstart Guide](https://github.com/eclipse/kuksa.val/blob/master/doc/quickstart.md)
+
+## Setup Mock Service (Optional)
+
+This step is only required when connecting to a local running Databroker, which is not integrated
+into an actual vehicle.
+
+To use the Companion App the Databroker needs to be connected to the Mock Service.
+The Mock Service simulates the interaction with the car e.g. interactions with the different
+interactors of a car. The steps to set up a local Databroker can be found in the
```

Comment:
maybe write "sensors and actuator" instead of duplicating interaction and interactors

Reply:
Done